success-criteria-problematic-for-closed-functionality.md

```diff
@@ -1,65 +1,65 @@
-<li><a href="#identify-input-purpose">1.3.5 Identify Input Purpose</a> — Requires information in a programmatically determinable form; in the absence of programmatic capabilities, text labels need to be specific and be provided to the user in other modalities (e.g. auditory).</li>
-<li><a href="#audio-control">1.4.2 Audio Control</a> — The intent of this success criterion is to avoid interference of audio with assistive products, which are not available in a system with closed functionality. But if the built-in accessibility features of the closed system provide speech output, then the interference may happen and this SC applies. In addition, there are existing closed functionality requirements in standards such as the EN 301 549 and U.S. Revised 508 Standards (402.3 Volume) that cover volume control for closed functionality products.</li>
-<li><a href="#contrast-minimum">1.4.3 Contrast (Minimum)</a> — There are cases where applying this success criterion to non-web software on closed functionality products is problematic:
+<li><a href="#info-and-relationships">1.3.1 Info and Relationships</a> — Requires that information be in a programmatically determinable form or in text (that is programmatically determinable).</li>
+<li><a href="#meaningful-sequence">1.3.2 Meaningful Sequence</a> — Requires information (i.e, a correct reading sequence) in a programmatically determinable form.  An equivalent for software with closed functionality would be to provide a meaningful reading sequence through auditory output or some other non-visual means that helps users correlate the output with the corresponding information displayed on the screen.</li>
+<li><a href="#orientation">1.3.4 Orientation</a> — Products with closed functionality that have fixed-in-place displays or other limitations to modifying the physical display orientation are covered under the essential exception and are not required to provide support for orientation changes.  See the note in the section <a href="#applying-sc-1-3-4-orientation-to-non-web-documents-and-software">Applying SC 1.3.4 Identify Input Purpose to Non-Web Documents and Software</a>.</li>
+<li><a href="#identify-input-purpose">1.3.5 Identify Input Purpose</a> — Depends upon information in a programmatically determinable form; in the absence of programmatic capabilities, text labels need to be specific and be provided to the user in other modalities (e.g. auditory).</li>
+<li><a href="#audio-control">1.4.2 Audio Control</a> — The intent of this success criterion is to avoid interference of audio with assistive products, which are not available in a system with closed functionality.  If the built-in accessibility features of a closed system provide speech output, then the interference may happen and this SC applies.  In addition, there are existing requirements in regulation (e.g., the EN 301 549 and U.S. Revised 508 Standards) that address volume control for products with closed functionality.</li>
```

Comment:
```suggestion
<li><a href="#audio-control">1.4.2 Audio Control</a> — The intent of this success criterion is to avoid interference of audio with assistive products, which are not available in a system with closed functionality.  If the built-in accessibility features of a closed system provide speech output, then the interference may happen and this SC applies.  In addition, there are existing requirements in regulations (e.g., the EN 301 549 and U.S. Revised 508 Standards) that address volume control for products with closed functionality.</li>
```